ownCloud/AppDelegate.swift

```diff
@@ -206,3 +208,25 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
 		}
 	}
 }
+
+import CrashReporter
```

Comment:
Wouldn't it be cleaner to add the import statement on the top of the file?

Reply:
@hosy moved but not sure if it really contributes to cleanliness